theano/sandbox/cuda/type.py

```diff
@@ -21,6 +21,11 @@
     import cuda_ndarray.cuda_ndarray as cuda
     from theano.sandbox.cuda.nvcc_compiler import NVCC_compiler
     import cuda_ndarray
+    # Python 3 does not necessarily set __file__. May need manual setting
+    try:
+        cuda_ndarray.__file__
+    except AttributeError:
+        cuda_ndarray.__file__ = os.path.join(cuda_ndarray.__path__._path[0], 'cuda_ndarray.pyd')
```

Comment:
cuda_ndarray.__path__._path don't exist in Python 2. But as this is in the except clause, it is fine.

Can you add in your comment that this happen in python 3 under windows and tell which python version/installation package you used?

This would explain people that would see this code why we use the pyd extension that is only for Windows. Also, I'm currious which stack on windows have this properties, as it is working in the case we tested recently.

As I didn't saw this problem on Linux, it is probably not useful to test if the file exist.
thanks